config.sh

```diff
@@ -15,13 +15,7 @@ function pre_build {
 
 function pip_opts {
     # Define extra pip arguments
-    echo "--prefer-binary --find-links $EXTRA_WHEELS_URL" 
-}
-
-function build_wheel {
```

Comment:
Why does build wheel disappear?

Reply:
Personally, I thought it might have been a relic of long ago. The default command seems to work well.

Reply:
I guess only @matthew-brett knows for sure.

Reply:
I reverted this for this PR.